doc(FEC 12554): create browsers support document + add missing player config documentation (#567)
kaltura/kaltura-player-js: docs/supportedPlatforms.md

## Kaltura Player Browsers Support 

> * Kaltura Player is being tested for playback & features on the stable and latest versions of the browsers below. 

Reviewer: ```suggestion
* Kaltura Player playback and features are tested on the stable and latest versions of the browsers below. 
```
